toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -257,3 +257,5 @@ audit-3.1.2-1.azl3.x86_64.rpm
 audit-libs-3.1.2-1.azl3.x86_64.rpm
 libxcrypt-4.4.36-2.azl3.x86_64.rpm
 libxcrypt-devel-4.4.36-2.azl3.x86_64.rpm
+ccache-4.8.3-1.azl3.x86_64.rpm
```

Comment:
This does not seem correct to me.

Reply:
It's for testing. I should've converted this PR to draft